### DISCUSSION TITLE
Maintainer needed?

### DISCUSSION THREAD

**Discussion by quank123wip** [wobsoriano/trpc-nuxt#172] [General]
Seems like this library is currently outdated as trpc has moved very fast. Maybe we need more PRs and contributors? Sadly, I don't know where to begin.

**Comment by wobsoriano** [comment]
It's hard to keep up with tRPC moving fast with v11 that's for sure, but luckily I had some time this weekend and published a beta for tRPC v11 support 😄 

`npm install @trpc/server@next @trpc/client@next trpc-nuxt@0.11.0-beta.1`

Always cool to have contributors so feel free to help!

**Reply by humblemodulo** [reply]
Any chance this fixes issues with Nuxt 4?  Using the "recommended" configuration (but adjusting to `app/plugins`) results in a `$client` that's merely a `Proxy` object with no useful features.

I can be more specific and log an actual issue if necessary.

**Reply by wobsoriano** [reply]
I havent really tried Nuxt 4 yet bull will do today. Any chance you could give me a basic repro? Then I can maybe play with that

**Reply by humblemodulo** [reply]
Wow, so creating a brand new repo, I discovered that there was an issue with naming on my part.  I'd changed the name to `trpcClient` in the `provide` section without giving any hint otherwise to Nuxt.  🤦‍♂️ 

**Everything appears to work now.**  Although it might be nice to simply use TRPC "directly" rather than go through NUXT plugins.  But I can wade through those waters on my own.

Thanks for letting me talk it through!  🤣
